client/js/libs/handlebars/colorClass.js

```diff
@@ -1,5 +1,3 @@
-"use strict";
-
```

Comment:
Oh well, I missed this (from http://eslint.org/docs/rules/strict):

> In ECMAScript modules, which always have strict mode semantics, the directives are unnecessary.

I'm wondering if ESLint's `strict` rule warns/errors if there is a `"use strict"` statement in these files, where there shouldn't be?